app/src/main/java/org/simple/clinic/overdue/Appointment.kt

```diff
@@ -138,5 +138,18 @@ data class Appointment(
 
     @Query("DELETE FROM Appointment")
     fun clear()
+
+    @Query("""
+      UPDATE Appointment SET status = :updatedStatus, syncStatus = :newSyncStatus, updatedAt = :newUpdatedAt
+      WHERE patientUuid = :patientUuid AND status = :scheduledStatus AND createdAt < :olderThan
```

Comment:
When I see the name `olderThan`, I expect the property to be a duration. If I were to write this behaviour down in english, it would be "Mark all scheduled appointments for a given patient and **older than** 13th March 2019 as visited", which sounds incorrect.

I would expect it to be more like "Mark all scheduled appointments for a given patient and **created before** 13th March 2019 as visited"

So, can you rename `olderThan` to something more appropriate like `createdBefore`?